README.md

```diff
@@ -24,6 +24,10 @@ Thank you for your interest in the firewatch project! Please find some informati
     - **Note:** If you get a notification on an issue, but would like to continue working on the issue without getting notifications, add the `ignore-passing-notification` label to the issue.
   - If firewatch config contains job success reporting, a jira will be created (with status `closed`) reporting the job success
 
+- Run a Gitleaks detect scan on the gathered artifacts.
+    - The latest Redhat detection rules are fetched from the [pattern distribution server](https://source.redhat.com/departments/it/it-information-security/wiki/pattern_distribution_server) and used by default.
+    - A JSON formatted report file is created if there are any detections.
```

Comment:
@amp-rh This may be something to bring up to the team, because I'm not really sure if this is something we want. It might be cool to add an optional functionality to write a bug to the default Jira project if results are found. 

CC: @ascerra

Reply:
If not, we can start with this and add it later as needed.

Reply:
OK, sounds good. I'll remove this for now.

Reply:
If we don't create a firewatch ticket for a security issue found after running gitleaks how would we know if we found an issue?

- We'd need to go into the output of each job and read through it to see if it found something?
- The way I see it currently, is that it is a requirement to report any issue found to Jira, I don't really see a case where we would find an issue and not report it.

let me know your thoughts